src/components/pages/import-batch-view.yml

```diff
@@ -0,0 +1,60 @@
+documentTitle:
+  en: Document
+  de: Dokument
+taskType:
+  en: Task type
+  de: Tasktyp
+taskStatus:
+  en: Task status
+  de: Taskstatus
+taskStatusPending:
+  en: Pending
+  de: Ausstehend
+taskStatusDone:
+  en: Done
+  de: Vertig
+createdOn:
+  en: Created on
+  de: Erstellt am
+completedOn:
+  en: Completed on
+  de: Vervollständigt am
+createdBy:
+  en: Created by
+  de: Erstellt von
+importHeading:
+  en: Import overview
+  de: Import ubersicht
```

Comment:
This guy is still wrong here.

Reply:
What the heck, I copy pasted it, hmmmmmmmmmm

Reply:
Fixed